### PR TITLE
Use Eslint form app.zetkin

### PR DESCRIPTION
**Pull request by amerharb**

### CODE REVIEW

File: webapp/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (richardolsson) [inline webapp/.eslintrc.js:1]: Could we use the same eslint config as the app.zetkin.org repo?

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Lint
```
Comment (richardolsson) [inline .github/workflows/lint.yaml:1]: Nice addition!

**Review by richardolsson — COMMENTED** [review]
Nice work! I wonder if we could use the same lint config as we do in our other repos (e.g. app.zetkin.org)?

File: webapp/.eslintrc.js (hunk 1 above)
Comment (amerharb) [reply webapp/.eslintrc.js:1]: I did that at the start but it doesn't work, something about prettier, it fail to read from prettier config....etc
so i just did the simple one
maybe if future we should make one repo for common config and lint that all repos imports

Comment (amerharb) [reply webapp/.eslintrc.js:1]: Now it has been included, how ever few eslint disable has been added to be fixed later

**Pull request merged**